pywr/_recorders.pyx

```diff
@@ -361,6 +361,79 @@ cdef class NumpyArrayNodeRecorder(NodeRecorder):
 NumpyArrayNodeRecorder.register()
 
 
+
+cdef class FlowDurationCurveRecorder(NumpyArrayNodeRecorder):
+    """
+    This recorder calculates a flow duration curve for each scenario.
+    The percentiles passed into the recorder must be in the range 0-100.
+    """
```

Comment:
We should add the arguments to the docstring. See `MeanFlowRecorder` for the agreed style. Some of the others in this module aren't correct.